### PR TITLE
fix(core): adjust payment related text

### PR DESCRIPTION
**Pull request by JimmFly**
work for #4785

image

### CODE REVIEW

File: packages/frontend/i18n/src/resources/en.json [hunk 1]
```diff
@@ -684,7 +684,7 @@
   "com.affine.auth.sign-out.confirm-modal.description": "After signing out, the Cloud Workspaces associated with this account will be removed from the current device, and signing in again will add them back.",
   "com.affine.auth.sign-out.confirm-modal.cancel": "Cancel",
   "com.affine.auth.sign-out.confirm-modal.confirm": "Sign Out",
-  "com.affine.payment.recurring-yearly": "Annually",
+  "com.affine.payment.recurring-yearly": "Yearly",
```
Comment (joooye34) [inline packages/frontend/i18n/src/resources/en.json:687]: I think the lower case should be default, upper case is special.

File: packages/frontend/core/src/components/affine/setting-modal/general-setting/plans/index.tsx [hunk 2]
```diff
@@ -21,13 +21,17 @@ import * as styles from './style.css';
 const getRecurringLabel = ({
   recurring,
   t,
+  shouldLowerCase,
 }: {
   recurring: SubscriptionRecurring;
   t: ReturnType<typeof useAFFiNEI18N>;
+  shouldLowerCase?: boolean;
```
Comment (joooye34) [inline packages/frontend/core/src/components/affine/setting-modal/general-setting/plans/index.tsx:28]: Use upperFirst from lodash to change this logic, let lower be default.

Comment (CatsJuice) [reply packages/frontend/core/src/components/affine/setting-modal/general-setting/plans/index.tsx:28]: Will it be better to implement with css only instead?

Comment (joooye34) [reply packages/frontend/core/src/components/affine/setting-modal/general-setting/plans/index.tsx:28]: oh, if css can resolve it, that is better, i don't know it.

**Review by joooye34 — APPROVED** [review]

**Pull request merged**